src/llm/bielik.py

```diff
@@ -0,0 +1,28 @@
+from langchain.chains.llm import LLMChain
+from langchain_ollama.llms import OllamaLLM
+from langchain_huggingface import HuggingFaceEmbeddings
+from langchain_core.prompts import PromptTemplate
+from langchain_community.vectorstores import FAISS
+
+
+class LLM:
+    def __init__(self, llm__model_name: str, template: str, embedding_model: HuggingFaceEmbeddings) -> None:
+        self.embedding_model = embedding_model

```

Comment:
I would separate this Embeddings model from LLM as they are 2 different models, so they can have separate classes and separate methods.